File: src/main/java/Entities/Coding.java
package Entities;

import org.json.JSONObject;

public class Coding {
	
	private static final String SYSTEM = "system";
	private static final String CODE = "code";
	private static final String DISPLAY = "display";
	private JSONObject entity;
	
	public Coding(String system, String code, String display) {
		entity = new JSONObject();
		entity.put(SYSTEM, system);
		entity.put(CODE, code);
		entity.put(DISPLAY, display);
	}
	
	public JSONObject getJSONObject() {
		return entity;
	}
}

File: src/main/webapp/js/Integration/Coding.js
function Coding(system, code, display) {
	this.system = system;
	this.code = code;
	this.display = display;
};

Coding.prototype.prettyPrint = function() {
	return "System: " + this.system
	+ " Code: " + this.code
	+ " Display: " + this.display;
};

Coding.prototype.jsonPrint = function() {
	return { system: this.system,
	code: this.code,
	display: this.display };
};

module.exports = Coding;
